_layouts/english.html

```diff
@@ -7,16 +7,24 @@
   <div class="container">
     <div class="row justify-content-end p-0 py-md-3 py-md-8">
       <div class="col-12 col-md-7 col-lg-5 p-3 p-md-3 p-lg-6" id="content">
-        <h2>2023 ZaiGeZaiGu Chinese New Year Gala Tickets On Sale Now</h2>
+        <h2>2023 ZGZG Summer Singing Contest
+        <p class="mt-1" style="font-size: 24px;">Open for registration!</p>
         <p>
-          <b>Date and Time: </b>Sat, January 21, 2023, 7PM – 10PM PST (New Year Festival 1-4PM) <br>
-          <b>Location: </b>San Jose Center for Performing Arts <br>
-          <b>Ticket Link: </b><a style="color: white;" href="https://zgzg.link/2023-ticket-website"><u>https://zgzg.link/2023-ticket-website</u></a><br>
-          <b>20%-off Promo code: </b> zgzg-early-bird (expires 01/08/23) <br>
+          Here, a spotlight shines just for you,<br>
+          A stage awaits your arrival,<br>
+          This will be a summer for you to achieve your musical dreams!<br><br>
+          Registration deadline: May 1st<br>
         </p>
-        <a href="https://zgzg.link/2023-ticket-website">
-          <button type="button" class="btn btn-light btn-primary btn-lg">Purchase Ticket</button>
-        </a>
+        <div style="display: flex; flex-direction: column; justify-content: flex-start; align-items: flex-start; gap: 10px">
+          <div style="display: flex; flex-wrap: wrap; justify-content: flex-start; align-items: flex-end; gap: 5px 10px;">
+            <a target="_blank" href="http://zgzg.link/2023s-signup">
+              <button type="button" class="btn btn-light btn-primary btn-sm" style="color: #0c4128;">Adult Group</button>
```

Comment:
style="color: #0c4128;":  since it's reused in multiple places, inline css may be not a good practice here. Better to extract it into scss files (Color is defined udner /assets/css/style.scss)
BTW, how did we decide which color to use?  Have you confirmed with Lizzi or designers(if we have)?

Reply:
add "btn-singer", it's only used for singing contest, because the theme color is green. we can just remove it when the even is end.